kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/ModalContainer.kt

```diff
@@ -49,21 +49,19 @@ abstract class ModalContainer<ModalRenderingT : Any>
   context: Context,
   attributeSet: AttributeSet? = null
 ) : FrameLayout(context, attributeSet) {
+  init {
+    // Stub base view will be replaced by registry.
+    this.addView(View(context))
+  }
 
-  private val base: View? get() = getChildAt(0)
+  private val baseView: View get() = getChildAt(0)
```

Comment:
This now seems over complicated, can we just do

```kotlin
private var baseView = View(context)
  .also { addView(it) }
```

And then the same assign-from-update below as for the `boardView` above?